hs_core/management/utils.py

```diff
@@ -28,6 +28,32 @@
 import logging
 
 
+def migrate_core_meta_elements(orig_meta_obj, comp_res):
+    """
+    Helper to migrate core metadata elements from the resource that is converted to
+    composite resource
+    :param orig_meta_obj: metadata object of the resource that is getting converted to
+    composite resource
+    :param comp_res: converted composite resource
+    :return:
+    """
+    single_meta_elements = ['title', 'type', 'language', 'rights', 'description',
+                            'publisher']
+    multiple_meta_elements = ['creators', 'contributors', 'coverages', 'subjects',
+                              'dates', 'formats', 'identifiers', 'sources', 'relations',
+                              'funding_agencies']
```

Comment:
'modified' in dates isn't getting copied over (calls to last_updated after the conversion is failing).

Reply:
I am not able to reproduce this. I am seeing the last updated date/time on UI after the conversion. I also see resource modified date in resourcemetadata.xml file of the converted resource.

Reply:
I think it is working now as I updated this with develop today.